frontend/apps/picasso/pages/transfers/index.tsx

```diff
@@ -43,7 +46,22 @@ const Transfers: NextPage = () => {
   const fee = useStore((state) => state.transfers.fee);
   const selectedToken = useStore((state) => state.transfers.selectedToken);
   const destFee = getDestChainFee(from, to, tokens, selectedToken);
-
+  const ksmBalance = useStore(
+    (state) => state.substrateBalances.balances.picasso.ksm.free
+  );
+  const isKSMAlertVisible = useMemo(
+    () =>
+      from === "picasso" &&
+      to === "statemine" &&
+      ksmBalance.lt(
+        fromChainIdUnit(
+          PICASSO_STATEMINE_KSM_TRANSFER_FEE,
+          tokens.ksm.decimals.picasso
+        )
+      ) &&
+      selectedToken === "usdt",
+    [from, to, ksmBalance, tokens.ksm.decimals.picasso, selectedToken]
+  );
```

Comment:
Does useMemo make sense here? On a quick skim it doesn't seem so to me

reference: https://blog.logrocket.com/rethinking-hooks-memoization/

Reply:
You are right, I usually don't overuse memoization, but had a gut feeling this one needed it. Will remove, retest and post update here.

Reply:
You got it 100%. it doesn't add any value. removing it.